app/controllers/draft_events_controller.rb

```diff
@@ -32,4 +32,42 @@ def join_draft
     }
     broadcast_message :join_draft, message, namespace: :drafts
   end
+
+  def pick_contestant
+    puts active_player == current_user
+    if active_player == current_user
+      puts 'in the right spot'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.